pkgs/by-name/fi/firefox-sync-client/package.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "firefox-sync-client";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Mikescher";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CNTTlBYJlQ4z6VXgi/mZpwxIAVOxJUnIL9flyvEzmXg=";
+  };
+
+  vendorHash = "sha256-Gb+4fxMBgvPB9Ki7zIwscY9l2kJ+tuE1Mc3W08YTfk8=";
+
+  meta = with lib; {
+    description = "Commandline-utility to list/view/edit/delete entries in a firefox-sync account.";
+    homepage = "https://github.com/Mikescher/firefox-sync-client";
+    license = licenses.asl20;
+    maintainers = [ maintainers.skamdart ];
+    mainProgram = "ffsclient";
+  };
+  passthru.updateScript = nix-update-script { };
```

Comment:
Please between vendorHash and meta, ideally with and empty line before and after. Putting it down here will effectively hide it, as meta is usually the last thing in a package.